Avoid using __typeof__ cause it is not supported by pycparser. Error
occurs on 32-bit systems.

--- a/setup.py
+++ b/setup.py
@@ -209,6 +209,7 @@ class type_generator(build_ext):
                         "-D__attribute__(x)=",
                         "-D__float128=long double",
                         "-D_FORTIFY_SOURCE=0",
+                        "-D_POSIX_SOURCE",
                     ],
                 )
                 parser = c_parser.CParser()
